src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams.py

```diff
@@ -112,151 +114,6 @@ def test_ams_storage_add_remove(self, resource_group, storage_account_for_create
             self.check('resourceGroup', '{rg}')
         ])
 
-    @ResourceGroupPreparer()
-    @StorageAccountPreparer(parameter_name='storage_account_for_create')
-    def test_ams_transform(self, resource_group, storage_account_for_create):
```

Comment:
Se borro por lagun motivo este test? Para hacerlo mas puntual según los casos de uso por así decirlo? Si no es así es raro que lo borremos

Reply:
I didn't delete this test, these changes came with the pull I performed on the saul-64 branch